blocks.js

```diff
@@ -7553,13 +7556,43 @@ InputSlotMorph.prototype.dropDownMenu = function (enableKeyboard) {
         if (Object.prototype.hasOwnProperty.call(choices, key)) {
             if (key[0] === '~') {
                 menu.addLine();
-            // } else if (key.indexOf('§_def') === 0) {
-            //     menu.addItem(choices[key].blockInstance(), choices[key]);
+         // } else if (key.indexOf('§_def') === 0) { 
+         //     menu.addItem(choices[key].blockInstance(), choices[key]);
+            } else if (key.charAt(key.length - 1) == '{') {
+                // Submenu
+                // 0x25ba = Unicode "BLACK RIGHT-POINTING POINTER"
+                // (not to be confused with "black right-pointing TRIANGLE")
+                menu.addItem(key.substring(0,key.length - 1).
+							 concat(String.fromCharCode(0x25ba)),
```

Comment:
Yeah, I don't have strong feelings about this one. :) (I think + is nice to read, but concat is definitely more type safe. 

> About the spacing, on the list for later is to right-justify the arrow.

oh this will be excellent. Then I'll go through and add keyboard shortcut icons. 😁